pimd/pim_vty.c

```diff
@@ -475,6 +475,8 @@ int pim_config_write(struct vty *vty, int writes, struct interface *ifp,
 			vty_out(vty, " " PIM_AF_NAME " pim dm\n");
 		else if (pim_ifp->pim_mode == PIM_MODE_SPARSE_DENSE)
 			vty_out(vty, " " PIM_AF_NAME " pim sm-dm\n");
+		else if (pim_ifp->pim_mode == PIM_MODE_SSM)
+			vty_out(vty, " " PIM_AF_NAME " pim ssm\n");
```

Comment:
for a long time, we only had sm mode, and so `ip pim` just meant both sm and ssm. I need to look again at the logic change I made with dm mode  before accepting this change or propose new changes.